lib/console.js

```diff
@@ -214,6 +219,15 @@ Console.prototype.countReset = function countReset(label = 'default') {
   counts.delete(`${label}`);
 };
 
+Console.prototype.group = function group() {
+  groupIndent++;
```

Comment:
This browsers this also prints whatever's passed to group:

![screenshot from 2017-08-18 14-48-56](https://user-images.githubusercontent.com/1538624/29447421-60329884-8424-11e7-9e70-8e420c9deb74.png)

IMO it should be something like:

```js
Console.prototype.group = function group(...args) {
  this.log(...args);
  this.groupIndent++;
};